### PR TITLE
[-] BO : update link preview in js when product change publish

### PR DESCRIPTION
**Pull request by cocoweb94**
## Description

change link preview when product change condition publish
## Steps to Test this Fix

1. change condition publish 
2. click on button preview 
3. if product is offline , link will have the token , admin dir and  id employee
## Forge Ticket (optional)

# BOOM-316

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -1210,12 +1210,18 @@ var form = (function() {
...
 
 			/** on active field change, send form */
-      $('#form_step1_active', elem).on('change', function() {
-        var active = $(this).prop('checked');
-        $('.for-switch.online-title').toggle(active);
-        $('.for-switch.offline-title').toggle(!active);
-        send();
-      });
+		    $('#form_step1_active', elem).on('change', function() {
+				var active = $(this).prop('checked');
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/js/bundle/product/form.js]: indentation please :)

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 2]
```diff
@@ -1008,6 +1008,7 @@
         </a>
         <a
           data-redirect="{{ preview_link }}"
+          data-url_deactive="{{ preview_link_deactivate }}"
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: I'd like to see `data-url-deactivate` instead, what do you think ?

File: src/Adapter/Product/AdminProductWrapper.php [hunk 3]
```diff
@@ -706,16 +705,33 @@ public function getPreviewUrl($product)
...
+     * @param string $preview_url
+     *
+     * @return string preview url deactivate
+     */
+    public function getPreviewUrlDeactivate($preview_url)
+    {
+        $context = \ContextCore::getContext();
+        $token = \ToolsCore::getAdminTokenLite('AdminProducts');
+
+        $admin_dir = dirname($_SERVER['PHP_SELF']);
```
Comment (mickaelandrieu) [inline src/Adapter/Product/AdminProductWrapper.php]: uhuh ^^ what do you expect ? did you take a look at the available constants in `defines.inc.php` file ?

**Comment by vincentbz** [comment]
Testé et ok, merci!

**Comment by mickaelandrieu** [comment]
Thank you @cocoweb94

**Pull request merged**